torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -113,13 +113,28 @@ graph(%a_quant, %b_quant, %scale, %zero_point, %dtype):
          %r = quantized::add_relu(%a_quant, %b_quant, %scale, %zero_point)
          return (%r) )";
 
+  std::string aten_linear = R"(
```

Comment:
Can you add a test for the pattern or move this PR into another that has a test?